src/common/RgnDetect.cpp

```diff
@@ -1987,7 +1991,12 @@ void CRgnDetect::PrepareTransparent(const CEFAR_INFO_MAPPING *apFarInfo, const C
                                     wchar_t* pChar, CharAttr* pAttr, int nWidth, int nHeight)
 {
 	_ASSERTE(pAttr!=mp_Attrs);
+	if (mb_SelfBuffers && mp_FarInfo)
+	{
+		free((CEFAR_INFO_MAPPING *)mp_FarInfo);
+	}
 	mp_FarInfo = apFarInfo;
```

Comment:
It is allocated in CRealBuffer::PrepareTransparent():

CEFAR_INFO_MAPPING* pFI = (CEFAR_INFO_MAPPING*)malloc(sizeof(CEFAR_INFO_MAPPING));